content/en/docs/v3.6/_index.md

```diff
@@ -0,0 +1,14 @@
+---
+title: v3.6.0-alpha docs
+cascade:
+  version: v3.6
+  versName: &name v3.6
+  git_version_tag: v3.6.0
+  exclude_search: false
+linkTitle: *name
+simple_list: true
+weight: -350 # Weight for doc version vX.Y should be -XY0
```

Comment:
```suggestion
weight: -360 # Weight for doc version vX.Y should be -XY0
```
This will sort this version on the top of the left hand menu.

Reply:
Good catch! Sounds good. Thanks @nate-double-u